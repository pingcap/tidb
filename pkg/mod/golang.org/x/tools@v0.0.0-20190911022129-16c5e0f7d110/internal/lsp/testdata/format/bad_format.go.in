package format //@format("package")

import (
	"runtime"
	"fmt"
	"log"
)

func hello() {




	var x int //@diag("x", "LSP", "x declared but not used")
}

func hi() {
	runtime.GOROOT()
	fmt.Printf("")

	log.Printf("")
}
