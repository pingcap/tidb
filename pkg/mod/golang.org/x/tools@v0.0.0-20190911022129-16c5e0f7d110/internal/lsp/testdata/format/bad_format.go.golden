-- gofmt --
package format //@format("package")

import (
	"fmt"
	"log"
	"runtime"
)

func hello() {

	var x int //@diag("x", "LSP", "x declared but not used")
}

func hi() {
	runtime.GOROOT()
	fmt.Printf("")

	log.Printf("")
}

-- gofmt-d --
--- format/bad_format.go.orig
+++ format/bad_format.go
@@ -1,16 +1,13 @@
 package format //@format("package")
 
 import (
-	"runtime"
 	"fmt"
 	"log"
+	"runtime"
 )
 
 func hello() {
 
-
-
-
 	var x int //@diag("x", "LSP", "x declared but not used")
 }
 

